.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Integration tests
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+          cache-dependency-path: |
+            setup.py
+            test-requirements.txt
+      - name: Install Python dependencies
+        run: |
+          pip install -r test-requirements.txt
+          pip install .
+      - name: Install RisingWave
+        run: curl -L https://risingwave.com/sh | sh
+      - name: Run integration tests
+        run: |
+          ./risingwave &
```

Comment:
This will pollute stdout, so better to redirect to a log file

Reply:
https://github.com/risingwavelabs/sqlalchemy-risingwave/pull/35